src/Components/person/Person.js

```diff
@@ -0,0 +1,45 @@
+import React, {Component} from 'react';
+import { Table } from 'semantic-ui-react';
+
+class Person extends Component {
+  render() {
+    const { person: { id, name, sex, age, born, died, century, father, mother, children },
+      onClick, selectedId } = this.props;
+
+    return (
+      <Table.Row
+        active={selectedId === id}
+        onClick={onClick} data-person-id={id}>
+        <Table.Cell>{id}</Table.Cell>
+        <Table.Cell
+          className={born < 1650
+          ? 'wasBornBefore1650'
+          : null}>
+          {name}
+        </Table.Cell>
+        <Table.Cell >{sex === 'm'
+          ? <span>&#128102;&#127999;</span>
+          : <span>&#128129;</span>}
+        </Table.Cell>
+        <Table.Cell
+          className={age >= 65
```

Comment:
```suggestion
          className={age >= 65 ? 'ageMore65' :  null}
        >
```
це може і в один рядок поміститись